MultiVersions/Forge/main/qualitycrops/lang/zh_tw.json

```
@@ -0,0 +1,122 @@
+  "item.qualitycrops.cookie_iron": "餅乾",
+  "item.qualitycrops.dough": "麵團",
+  "item.qualitycrops.dough_diamond": "麵團",
+  "item.qualitycrops.dough_gold": "麵團",
+  "item.qualitycrops.dough_iron": "麵團",
+  "item.qualitycrops.dried_kelp_diamond": "海帶乾",
+  "item.qualitycrops.dried_kelp_gold": "海帶乾",
+  "item.qualitycrops.dried_kelp_iron": "海帶乾",
+  "item.qualitycrops.egg_diamond": "雞蛋",
+  "item.qualitycrops.egg_gold": "雞蛋",
+  "item.qualitycrops.egg_iron": "雞蛋",
+  "item.qualitycrops.glow_berries_diamond": "螢光莓",
+  "item.qualitycrops.glow_berries_gold": "螢光莓",
+  "item.qualitycrops.glow_berries_iron": "螢光莓",
+  "item.qualitycrops.kelp_diamond": "海帶",
+  "item.qualitycrops.kelp_gold": "海帶",
+  "item.qualitycrops.kelp_iron": "海帶",
+  "item.qualitycrops.melon_slice_diamond": "西瓜片",
+  "item.qualitycrops.melon_slice_gold": "西瓜片",
+  "item.qualitycrops.melon_slice_iron": "西瓜片",
+  "item.qualitycrops.mushroom_stew_diamond": "蘑菇湯",
+  "item.qualitycrops.mushroom_stew_gold": "蘑菇湯",
+  "item.qualitycrops.mushroom_stew_iron": "蘑菇湯",
+  "item.qualitycrops.mutton_diamond": "生羊肉",
+  "item.qualitycrops.mutton_gold": "生羊肉",
+  "item.qualitycrops.mutton_iron": "生牛肉",
+  "item.qualitycrops.poisonous_potato_diamond": "毒馬鈴薯",
+  "item.qualitycrops.poisonous_potato_gold": "毒馬鈴薯",
+  "item.qualitycrops.poisonous_potato_iron": "毒馬鈴薯",
+  "item.qualitycrops.porkchop_diamond": "生豬肉",
+  "item.qualitycrops.porkchop_gold": "生豬肉",
+  "item.qualitycrops.porkchop_iron": "生豬肉",
+  "item.qualitycrops.potato_diamond": "馬鈴薯",
+  "item.qualitycrops.potato_gold": "馬鈴薯",
+  "item.qualitycrops.potato_iron": "馬鈴薯",
+  "item.qualitycrops.pufferfish_diamond": "河豚",
+  "item.qualitycrops.pufferfish_gold": "河豚",
+  "item.qualitycrops.pufferfish_iron": "河豚",
+  "item.qualitycrops.pumpkin_pie_diamond": "南瓜派",
+  "item.qualitycrops.pumpkin_pie_gold": "南瓜派",
+  "item.qualitycrops.pumpkin_pie_iron": "南瓜派",
+  "item.qualitycrops.rabbit_diamond": "生兔肉",
+  "item.qualitycrops.rabbit_gold": "生兔肉",
+  "item.qualitycrops.rabbit_iron": "生兔肉",
+  "item.qualitycrops.rabbit_stew_diamond": "兔肉湯",
+  "item.qualitycrops.rabbit_stew_gold": "兔肉湯",
+  "item.qualitycrops.rabbit_stew_iron": "兔肉湯",
+  "item.qualitycrops.red_mushroom_diamond": "紅色蘑菇",
+  "item.qualitycrops.red_mushroom_gold": "紅色蘑菇",
+  "item.qualitycrops.red_mushroom_iron": "紅色蘑菇",
+  "item.qualitycrops.salmon_diamond": "生鮭魚",
+  "item.qualitycrops.salmon_gold": "生鮭魚",
+  "item.qualitycrops.salmon_iron": "生鮭魚",
+  "item.qualitycrops.sweet_berries_diamond": "甜莓",
+  "item.qualitycrops.sweet_berries_gold": "甜莓",
+  "item.qualitycrops.sweet_berries_iron": "甜莓",
+  "item.qualitycrops.tropical_fish_diamond": "熱帶魚",
+  "item.qualitycrops.tropical_fish_gold": "熱帶魚",
+  "item.qualitycrops.tropical_fish_iron": "熱帶魚",
+  "item.qualitycrops.wheat_diamond": "小麥",
+  "item.qualitycrops.wheat_gold": "小麥",
+  "item.qualitycrops.wheat_iron": "小麥",
+  "itemGroup.qualitycrops.crops": "優質作物",
+  "itemGroup.qualitycrops.items": "優質作物物品"
```

當模組在創造模式中，有多個物品群組時，我們會習慣在模組名稱後，使用 `｜` 
```suggestion
  "itemGroup.qualitycrops.items": "優質作物｜物品"
```